eureka-client/src/main/java/com/netflix/discovery/shared/transport/decorator/RetryableEurekaHttpClient.java

```diff
@@ -126,7 +126,7 @@ public void shutdown() {
                 }
                 logger.warn("Request execution failure with status code {}; retrying on another server if available", response.getStatusCode());
             } catch (Exception e) {
-                logger.warn("Request execution failed with message: {}", e.getMessage());  // just log message as the underlying client should log the stacktrace
+                logger.warn("Request execution failed with message: {}", e.getMessage(), e);  // just log message as the underlying client should log the stacktrace
```

Comment:
The full stacktrace is explicitly not logged here (see comment).

Reply:
Oops, I misread the comment. Sorry about that!

While rebasing to resolve the conflict with d8acdba71706ded8dd5613f98936211d471c6f16, I dropped the "Also log the Exception" commit.